src/routes/blog/spyder-6-remote-development/+page.md

```diff
@@ -0,0 +1,94 @@
+
+Alongside built-in backend and client-server components, Spyder 6.0 features a new remote connection manager that allows you to connect to any machine with a SSH server, no prior installation or configuration needed, and automatically set up and use a remote Python environment  on the host.
+It also features graphical status checking and reporting, host configuration management, and more.
+In addition, it brings a new `Consoles` submenu allowing you to connect to any remote environment with one click, launch a console and run your code.
+You can also stop code execution as well as restart remove kernels from right within Spyder.
+
+<video controls>
+  <source src="remote-console-demo.mp4" type="video/mp4">
+  <track kind="captions">
+  Sorry, your browser doesn’t support HTML5 video. Download the <a href="screencast.mp4">MP4 file</a>
+</video>
+
+We're excited to have our newest core developer, Hendrik Louzada, and our team share with you their insights and perspectives on Spyder's new remote development platform introduced in Spyder 6!
+Join us as Hendrik shares how he got started with Spyder and the project, how the new achetecture is implemented under the hood, what challenges we faced and what he learned from them, and what's next for Spyder 6.1.0!
+
+
+## How'd you find out about Spyder and why'd you join the project?
+
+Hendrick first started to work on Spyder around three years ago, developing a series of plugins to control remote magnetic resonance equipment (such as MRI machines) directly in the IDE.
+As part of that project, he made multiple upstream contributions to Spyder itself to improve the Language Server Protocol support (LSP, the architecture that powers Spyder's code completion, introspection, analysis and formatting).
+
+In 2023, Spyder received a [Chan Zuckerberg Initiative](https://chanzuckerberg.com/) (CZI) [EOSS 5 grant](https://chanzuckerberg.com/eoss/proposals/enhancing-spyder-ide-remote-support-for-scientific-research-in-python/) to implement a new remote development platform and features in Spyder to allow users to develop and run code in remote desktops, servers, clusters and cloud machines.
+After a six-month search by Spyder lead maintainer Carlos Cordoba to find the right candidate to engineer the backend and network architecture for this project, he found Hendrik, who joined the core development team near the end of 2023.
+
+
+![New remote connection manager dialog in Spyder listing the configurable settings for a new remote host](remote-connection-manager-new.png)
+
+## How is the new remote development platform implemented?
+
+The backend, [`spyder-remote-services`](https://github.com/spyder-ide/spyder-remote-services), uses [Jupyter Server](https://jupyter-server.readthedocs.io/) and the [JupyterHub API](https://jupyterhub.readthedocs.io/en/stable/reference/rest-api.html), since Spyder is already leverages the Jupyter architecture to manage kernels and run code.
```

Comment:
See rationale above for why this formal structure wouldn't be necessary or appropriate in this context.

```suggestion
The backend, [`spyder-remote-services`](https://github.com/spyder-ide/spyder-remote-services), uses [Jupyter Server](https://jupyter-server.readthedocs.io/) and the [JupyterHub API](https://jupyterhub.readthedocs.io/en/stable/reference/rest-api.html), since Spyder already leverages the Jupyter architecture to manage kernels and run code.
```